cartography/data/indexes.cypher

```diff
@@ -191,3 +191,14 @@ CREATE INDEX ON :AzureVirtualMachine(id);
 CREATE INDEX ON :AzureDataDisk(id);
 CREATE INDEX ON :AzureDisk(id);
 CREATE INDEX ON :AzureSnapshot(id);
+CREATE INDEX ON :KubernetesCluster(id);
+CREATE INDEX ON :KubernetesCluster(name);
+CREATE INDEX ON :KubernetesNamespace(id);
+CREATE INDEX ON :KubernetesNamespace(name);
+CREATE INDEX ON :KubernetesPod(id);
+CREATE INDEX ON :KubernetesPod(name);
+CREATE INDEX ON :KubernetesContainer(id);
+CREATE INDEX ON :KubernetesContainer(name);
+CREATE INDEX ON :KubernetesContainer(image);
+CREATE INDEX ON :KubernetesService(id);
+CREATE INDEX ON :KubernetesService(name);
```

Comment:
It's more of an added feature as mostly humans will be more interested in querying on the basis of `name` rather than `id` (or UUID) while doing scripting queries based on `id` make more sense.

Reply:
`name` is required, since as Rana mentioned, when people are querying they'll use name, and without an index here, these queries may never finish. Note this is a common pattern elsewhere in this file for the same reason.

Reply:
I see, ok. @achantavy Could you give this a quick look and let me know if this is alright?

Reply:
Yup this is fine